accord-core/src/test/java/accord/utils/Property.java

```diff
@@ -944,6 +944,19 @@ public CommandsBuilder<State, SystemUnderTest> addIf(Predicate<State> predicate,
             return this;
         }
 
+        @SafeVarargs
+        public final CommandsBuilder<State, SystemUnderTest> addAllIf(Predicate<State> predicate,
+                                                                      Setup<State, SystemUnderTest> first,
+                                                                      Setup<State, SystemUnderTest> second,
+                                                                      Setup<State, SystemUnderTest>... reset)
```

Comment:
the reason for this style is to force you to use `addIf` when there is only 1 `Setup`, so can only call if-and-only-if there are at least 2.